frontend/src/store/actions.js

```diff
@@ -64,4 +67,25 @@ export const actions = {
       .then(isActive => commit(mutations.SET_VERIFICATION_TOKEN_ACTIVE, { token, isActive }))
       .then(() => commit(mutations.SET_VERIFICATION_TOKEN_SUBMISSION_PROGRESS, false));
   },
+  async [SYNC]({ dispatch, state, commit }) {
+    if (state.routes.length === 0 && state.user) {
+      dispatch(`${namespaces.drives}/${apiActions.fetchDrives}`);
+      return;
+    }
+
+    if (state.routes.length === 0 || !state.user || !navigator.onLine) {
+      return;
+    }
+
```

Comment:
Empty line but who cares.

Reply:
we can add this to linter if that is an error

Reply:
but yeah - look weird